source/EduCATS/Pages/Pickers/SubjectsViewModel.cs

```diff
@@ -138,7 +138,7 @@ async Task<IList<SubjectModel>> getSubjects()
 		/// <summary>
 		/// Setup subject by subject name.
 		/// </summary>
-		void setupSubject()
+		public void setupSubject()
```

Comment:
Public methods should start with uppercase.
Private methods should start with lowercase.
Try to follow these rules.